tests/components/magichome/test_init.py

```diff
@@ -0,0 +1,563 @@
+"""The tests for the Light component."""
+# pylint: disable=protected-access
+import unittest
+import unittest.mock as mock
+import os
+from io import StringIO
+
+import pytest
+
+from homeassistant import core
+from homeassistant.component import magichome as MagicHomeApi
```

Comment:
Just a small typo: `components` instead of `component` 😉

Reply:
OK！ Thank